src/fklearn/training/imputation.py

```diff
@@ -32,23 +33,41 @@ def imputer(df: pd.DataFrame,
         - If "mean", then replace missing values using the mean along the axis.
         - If "median", then replace missing values using the median along the axis.
         - If "most_frequent", then replace missing using the most frequent value along the axis.
+
+    fill_value : Any, (default=None)
+        if not None, use this as default value when some feature only contains NA values.
```

Comment:
If this is the default value, I would change this name to something like `default_fill_value` or something like this

Reply:
Also, I would use the `column` instead of `feature` and `missing` instead of `NA`:

```
if not None, use this as default value when one of the columns_to_impute only contains missing values.
```